mscore/prefsdialog.ui

```diff
@@ -2779,6 +2779,68 @@
             </property>
            </widget>
           </item>
+          <item row="2" column="2">
+           <widget class="QLabel" name="portMidiOutputLabel">
+            <property name="text">
+             <string>MIDI Output:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="2" column="4">
+           <widget class="QComboBox" name="portMidiOutput">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="minimumSize">
+             <size>
+              <width>0</width>
+              <height>25</height>
+             </size>
+            </property>
+            <property name="accessibleName">
+             <string>MIDI Output</string>
+            </property>
+            <property name="accessibleDescription">
+             <string>Choose MIDI Output</string>
+            </property>
+           </widget>
+          </item>
+          <item row="2" column="6">
+           <layout class="QHBoxLayout" name="horizontalLayout_5">
+            <item>
+             <widget class="QLabel" name="portMidiOutputLatencyLabel">
+              <property name="text">
+               <string>MIDI Output Latency:</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QSpinBox" name="portMidiOutputLatency">
+              <property name="toolTip">
+               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Latency&lt;/span&gt; is a delay in milliseconds applied to timestamps, to inform the driver about when the output should actually occur.&lt;/p&gt;&lt;p&gt;If &lt;span style=&quot; font-weight:600;&quot;&gt;latency&lt;/span&gt; is zero, MIDI messages are delivered as fast as possible, but may contain jitter.&lt;/p&gt;&lt;p&gt;Setting &lt;span style=&quot; font-weight:600;&quot;&gt;latency&lt;/span&gt; greater than zero may help the driver alleviate any jitter.&lt;/p&gt;&lt;p&gt;Adjusting &lt;span style=&quot; font-weight:600;&quot;&gt;latency&lt;/span&gt; can help synchronize your midi hardware with MuseScore's internal audio synthesizer, so that they both sound notes at the same time.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              </property>
+              <property name="accessibleName">
+               <string>MIDI Output Latency</string>
+              </property>
+              <property name="accessibleDescription">
+               <string>Choose MIDI Output Latency</string>
+              </property>
+              <property name="suffix">
+               <string> ms</string>
```

Comment:
Maybe  better `<string extracomment="milliseconds">ms</string>` like in other ports of that dialog